container/setup/setup-container.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# test if ES version supplied
+if [ -z $1 ]; then
+   ES_VERSION='"2.0.1.0"'
+else
+   ES_VERSION=\"${1}\"
+fi
+
+# set user volume
+USER_VOLUME=/root/user_volume
+
+# set the version variables
+SPARK_VERSION=$(jq -r .release.${ES_VERSION}.sparkversion $USER_VOLUME/es-releases.json)
+SPARK_CLIENT=$(jq -r .release.${ES_VERSION}.sparkclient $USER_VOLUME/es-releases.json)
+
+# Setup java/scala/python/kafka/rest/spark
+${SETUP_AREA}/setup-java.sh
+${SETUP_AREA}/setup-scala.sh
+${SETUP_AREA}/setup-python.sh
+${SETUP_AREA}/setup-kafka.sh
```

Comment:
I think more work is needed for kafka here, recall that the KAFKA repo actually stores a hard-coded es-client JAR i.e. here:
https://github.com/IBMProjectEventStore/db2eventstore-kafka/tree/master/event-stream/lib

We did add a tag for the older release:
https://github.com/IBMProjectEventStore/db2eventstore-kafka/tags

So `setup-kafka` would need to also be passed the correct version to clone.

FYI feel free to handle this as a *follow on* issue.

Reply:
I think with the sprint almost done it makes sense to make Kafka, and the other apps es version aware